`invoker` fails when arity is 0

What I see:

The following test case
```python
def test_open_cursor(db_context):
    class TestClass:
        def one(self):
            return 1

    one = R.invoker(0, "one")(TestClass())
    assert one == 1
```
fails with 
```
arity = 0, f = 'one'

    @curry
    def invoker(arity, f):
        """Turns a named method with a specified arity into a function that can be
        called directly supplied with arguments and a target object.
        The returned function is curried and accepts arity + 1 parameters where
        the final parameter is the target object"""
        _args = ", ".join([f"x{i}" for i in range(0, arity)])
        _f = f'lambda {_args}, object: getattr(object, "{f}")({_args})'
>       return curry(eval(_f))
E         File "<string>", line 1
E           lambda , object: getattr(object, "one")()
E                  ^
E       SyntaxError: invalid syntax
```

What I would expect:
`R.invoker(0, 'method_name')(class_instance)` calls `method_name` on `class_instance` without any arguments.

--- a/ramda/invoker.py
+++ b/ramda/invoker.py
@@ -7,6 +7,6 @@ def invoker(arity, f):
     called directly supplied with arguments and a target object.
     The returned function is curried and accepts arity + 1 parameters where
     the final parameter is the target object"""
-    _args = ", ".join([f"x{i}" for i in range(0, arity)])
-    _f = f'lambda {_args}, object: getattr(object, "{f}")({_args})'
+    _args = "".join([f"x{i}, " for i in range(0, arity)])
+    _f = f'lambda {_args} object: getattr(object, "{f}")({_args})'
     return curry(eval(_f))

--- a/ramda/invoker_test.py
+++ b/ramda/invoker_test.py
@@ -7,3 +7,10 @@ def invoker_test():
     assert_equal(replacer("abcdef", "", "abcdefghijklm"), "ghijklm")
     replace_all = invoker(2, "replace")("abcdefghijklm")
     assert_equal(replace_all("gh", "abcdefghijklm"), "gh")
+
+    class TestClass:
+        def one(self):
+            return 1
+
+    one = invoker(0, "one")(TestClass())
+    assert one == 1

--- a/setup.py
+++ b/setup.py
@@ -24,7 +24,7 @@ setup(
     author_email="slava.ganzin@gmail.com",
     packages=["ramda", "ramda.private", "ramda.private.curry_spec"],
     install_requires=["future", "toolz"],
-    tests_require=["nose", "coverage", "mock"],
+    tests_require=["nose", "coverage", "mock", "six"],
     cmdclass={"build_py": PasteurizeBuildCommand},
     long_description=long_description,
     long_description_content_type="text/markdown",

